src/components/ArmEilrExamples.svelte

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+  import { page } from "$app/stores";
+  import { buildHyperlink } from "../helpers/buildHyperlinkHelper";
+  import { selected } from "../stores/selected";
+  export let reverseType = false;
+
+  $: ewNotSelected = $selected && $selected.geoType != "ew";
```

Comment:
In JS you always use strict equality, `!==` and `===`

Reply:
Also can use Optional Chaining operator for expressions likes this, like `$selected?.geoType !== "ew"`

Reply:
fixed the operator